features/makepot.feature

```diff
@@ -1615,6 +1615,11 @@ Feature: Generate a POT file of a WordPress project
 
       Object(u.__)( 'minified.__', 'foo-plugin' );
       Object(j._x)( 'minified._x', 'minified._x_context', 'foo-plugin' );
+
+      /* translators: babel */
```

Comment:
Nit: could test whether the comment is extracted successfully

Reply:
Thanks, just added that 👍